### ISSUE TITLE
Smooth creation of Reinforced Concrete Sections

### ISSUE THREAD

**Issue by DanielGMorenaFhecor** [fib-international/structuralcodes#136]
Creating rectangular reinforced concrete sections is a common task. It would be convenient to have a utility method and a `__add__` operator to create something like this: `section.rectangular(b, h, mat) + steel_reinf(...) + steel_reinf(...) + ...`.  

If this is not the best approach or is difficult to implement (I haven’t explored this section of the code deeply), perhaps a utility function or wrapper could be a good alternative.  

This is not a critical task but would improve usability for those evaluating the ease of use of this library.

**Comment by talledodiego** [comment]
You are right. I started developing a code in this direction in the last months. 
Then Simen took over and he is currently working on this.

yes we can use `__add__` as you suggest (for sure it will be permitted to the user), but what we are developing is actually:
a `rectangular` section that represents exactly what you are saying (so a rectangle of a homogeneous material). The rectangular section will be created as you suggest: `section.rectangular(b, h, mat)`.

Furthermore we added also a `RectangularRC` section that will permit to input standard rectangular RC sections even easier.

As soon as Simen creates the PR (now the code is in its own fork) I will link it here so that we can discuss further.
